invision-app/app/components/home/home.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { ErrorBoundary } from 'react-error-boundary';
+import { AddMovie} from '../addMovie'
+import { MovieFiterAndSort } from '../movieFiterAndSort/movieFiterAndSort';
```

Comment:
You have index file in the structure of this component. It means that you can import this component: ```import { MovieFiterAndSort } from '../movieFiterAndSort';```. Also, please look at the [**alias**](https://webpack.js.org/configuration/resolve/#resolvealias) webpack feature that allows you insert all components more understandable: ```import { MovieFiterAndSort } from 'Components/movieFiterAndSort';```

Reply:
Done